docs/kyma/docs/040-cr-installation.md

```diff
@@ -26,14 +26,30 @@ metadata:
 spec:
   version: "1.0.0"
   url: "https://sample.url.com/kyma_release.tar.gz"
+  components: 
+    - name: "cluster-essentials"
+      namespace: "kyma-system"
+    - name: "istio"
+      namespace: "istio-system"
+    - name: "prometheus-operator"
+      namespace: "kyma-system"
+    - name: "provision-bundles"
+    - name: "dex"
+      namespace: "kyma-system"
+    - name: "core"
+      namespace: "kyma-system"
 ```
 
 This table lists all the possible parameters of a given resource together with their descriptions:
 
 | Field   |      Mandatory?      |  Description |
 |:----------:|:-------------:|:------|
 | **metadata.name** | **YES** | Specifies the name of the CR. |
-| **metadata.labels.action** | **YES** | Defines the behavior of the Kyma installer. Available options are `install` and `uninstall`. |
+| **metadata.labels.action** | **YES** | Defines the behavior of the Kyma installer. Available options are `install`, `update` and `uninstall`. |
 | **metadata.finalizers** | **NO** | Protects the CR from deletion. Read [this](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers) Kubernetes document to learn more about finalizers. |
 | **spec.version** | **NO** | When manually installing Kyma on a cluster, specify any valid [SemVer](https://semver.org/) notation string. |
 | **spec.url** | **YES** | Specifies the location of the Kyma sources `tar.gz` package. For example, for the `master` branch of Kyma, the address is `https://github.com/kyma-project/kyma/archive/master.tar.gz` |
+| **spec.components** | **YES** | List of Helm charts components to `install` or `update` |
+| **spec.components.name** | **YES** | The name of the component is also the name of the component subdirectory in the `resources` directory. |
+| **spec.components.namespace** | **YES** | Namespace of the component in which it will be installed or updated. |
+| **spec.components.release** | **NO** | The name of Helm release in case it should differ from component name. |
```

Comment:
`Provides the name of the Helm release in case it should differ from the component name.`